### ISSUE TITLE
Running the converter from the command line

### ISSUE THREAD

**Issue by mskyttner** [seinecle/gexf-vosviewer-converter#2]
It would be nice if it would be possible to execute the jar from the command line to convert a .json or .gexf.

**Comment by seinecle** [comment]
For automation purposes? If so, would a web API be convenient to you as well?

**Comment by mskyttner** [comment]
Yes, my intended use would be for automation purposes from the command line, non-interactive, non-networked. For example when making a bash script to convert vosviewer-online-json files to gexf and vice versa.

For me personally, primarily a command variant exposing the conversion methods in the library would be convenient as it would enable usage from any programming language or script. If there was a jar/war that could expose a Web API that could be nice too, provided it was possible to run it from a container (ie as a self-hosted service).
